manifest-json-schema.json

```diff
@@ -67,9 +67,9 @@
       "description": "If true, run OTP as a server.",
       "default": true
     },
-    "s3UploadBucket": {
+    "s3UploadPath": {
       "type": "string",
-      "description": "The base path of an s3 bucket where files will be uploaded to."
+      "description": "The base path of an s3 bucket where files will be uploaded to. Ex: `s3://path/to/folder`"
```

Comment:
Very nitpicky, sorry, but could you change example path to `s3://bucket-name/folder`? I think making it clear that it contains the bucket name is important.